sql/sys_vars.cc

```diff
@@ -1570,6 +1570,15 @@ Sys_log_bin_compress_min_len(
   GLOBAL_VAR(opt_bin_log_compress_min_len),
   CMD_LINE(OPT_ARG), VALID_RANGE(10, 1024), DEFAULT(256), BLOCK_SIZE(1));
 
+static Sys_var_enum Sys_binlog_error_action(
+       "binlog_error_action",
+       "When statements cannot be written to the binary log due to a fatal "
+       "error, the server can either ignore the error and let the primary "
+       "continue, or abort",
+       GLOBAL_VAR(binlog_error_action), CMD_LINE(REQUIRED_ARG),
+       (const char *[]){"IGNORE_ERROR", "ABORT_SERVER", NullS},
+       DEFAULT(IGNORE_ERROR));
+
 static Sys_var_on_access_global<Sys_var_mybool,
```

Comment:
What’s a `Sys_var_on_access_global`?

Reply:
Its basically an authentication wrapper to secure the variable s.t. its modification is protected by some role (see `set_var::check()` and for more details on the variable checking process).